app/views/govuk_publishing_components/components/docs/input.yml

```
@@ -57,6 +57,20 @@ examples:
       name: "labelledby"
       describedby: "wrapper"
       controls: "wrapper"
+  with_password_reveal:
+    description: |
+      Requires that the input type is also set to `password`.
+
+      Adds a password reveal button using JS after the input, which toggles the type of the input from password to text, revealing the password. It also sets the autocomplete on the input to "off", so that browsers don't accidentally remember the password.
+
+      Uses a visually hidden bit of text to inform screen reader users of the state of the input, in preference to announcing the content of the password input when toggled.
```

Small rewording suggestion - I found the description a little hard to understand, so attempted to make it a bit clearer.

```suggestion
      Adds a password reveal button after the input which toggles the type of the input from password to text revealing the password. It also sets the `autocomplete` attribute on the input to "off" so that browsers don't accidentally remember the password.

      Uses a visually hidden bit of text to inform screen reader users of the state of the input rather than announcing the content of the password input when toggled.
```

---

Thanks, yeah that does read better. Changed.